<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>471</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>812</width>
    <height>471</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>812</width>
    <height>471</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="TimeTableQt.qrc">
    <normaloff>:/TimeTableQt/123.ico</normaloff>:/TimeTableQt/123.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_16">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QListWidget" name="listView">
         <property name="minimumSize">
          <size>
           <width>246</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_delete">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_order">
         <property name="text">
          <string>排序</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="QWidget" name="tab_lessons">
          <attribute name="title">
           <string>课程</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QComboBox" name="comboBox_LessonDays">
                 <item>
                  <property name="text">
                   <string>Mon</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Tue</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Wed</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Thu</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Fri</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Sat</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Sun</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox">
                 <property name="text">
                  <string>仅显示当天</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QComboBox" name="comboBox_addLesson">
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                 <property name="currentText">
                  <string>语文</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>语文</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>数学</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>英语</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>物理</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>化学</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>生物</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>历史</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>政治</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>地理</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>体育</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>艺术</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>信息技术</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="timeEdit_begin">
                 <property name="time">
                  <time>
                   <hour>6</hour>
                   <minute>0</minute>
                   <second>0</second>
                  </time>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="timeEdit_end">
                 <property name="time">
                  <time>
                   <hour>22</hour>
                   <minute>0</minute>
                   <second>0</second>
                  </time>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QPushButton" name="pushButton_changeLesson">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>修改</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_addLesson">
                 <property name="text">
                  <string>添加</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_infos">
          <attribute name="title">
           <string>信息</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QComboBox" name="comboBox_InfoDays">
                 <property name="currentText">
                  <string>Mon</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>Mon</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Tue</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Wed</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Thu</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Fri</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Sat</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Sun</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>仅显示当天</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <property name="tristate">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_22">
               <item>
                <widget class="QLineEdit" name="lineEdit_changeInfo"/>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="pushButton_changeInfo">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>修改</string>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_addInfo">
                 <property name="text">
                  <string>添加</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_windowsettings">
          <attribute name="title">
           <string>窗口</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>窗口位置</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_windowLocationX">
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_windowLocationY">
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_19">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>窗口大小</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_windowSizeX">
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_windowSizeY">
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>无课程时显示</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_lessonNull"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>位于第</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox_lessonInLine"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>项</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>倒数日</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateTimeEdit" name="dateTimeEdit"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>位于第</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox_DayCountInLine"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>项</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="QLineEdit" name="lineEdit_backGroundImg"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_chooseBackGround">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_applySettings">
             <property name="text">
              <string>应用</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_textFormat">
          <attribute name="title">
           <string>文字项</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>文字位置</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_ItemLocationX">
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_ItemLocationY">
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_color">
                 <property name="whatsThis">
                  <string>color</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_chooseColor">
                 <property name="text">
                  <string>选择</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_textFormat"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QSpinBox" name="spinBox_FontSize">
                 <property name="value">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFontComboBox" name="fontComboBox_textFont"/>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_21">
               <item>
                <widget class="QPushButton" name="pushButton_addFormat">
                 <property name="text">
                  <string>添加</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_changeFormat">
                 <property name="text">
                  <string>修改</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_configPath">
          <attribute name="title">
           <string>配置文件</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>配置文件路径</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLineEdit" name="lineEdit_ConfigPath"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_chooseConfig">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_applyConfigPath">
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>课程信息路径</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLineEdit" name="lineEdit_LessonPath"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_chooseLessons">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_applyLessonPath">
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_close">
         <property name="text">
          <string>关闭</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="TimeTableQt.qrc"/>
 </resources>
 <connections/>
</ui>
